src/main/java/io/polivakha/mojo/properties/WriteActiveProfileProperties.java

```diff
@@ -31,30 +31,25 @@
  * Writes properties of all active profiles to a file.
  *
  * @author <a href="mailto:zarars@gmail.com">Zarar Siddiqi</a>
+ * @author Sergey Korotaev
  */
-@Mojo( name = "write-active-profile-properties", defaultPhase = LifecyclePhase.NONE, threadSafe = true )
-public class WriteActiveProfileProperties
-    extends AbstractWritePropertiesMojo
-{
-    /** {@inheritDoc} */
+@Mojo(name = "write-active-profile-properties", defaultPhase = LifecyclePhase.NONE, threadSafe = true)
+public class WriteActiveProfileProperties extends AbstractWritePropertiesMojo {
+    /**
+     * {@inheritDoc}
+     */
     public void execute()
-        throws MojoExecutionException
-    {
+            throws MojoExecutionException {
         validateOutputFile();
-        List<Profile> list = getProject().getActiveProfiles();
-        if ( getLog().isInfoEnabled() )
-        {
-            getLog().debug( list.size() + " profile(s) active" );
+        List<Profile> profileList = getProject().getActiveProfiles();
+        if (getLog().isInfoEnabled()) {
+            getLog().debug(profileList.size() + " profile(s) active");
         }
         Properties properties = new Properties();
-        for ( Profile profile : list )
-        {
```

Comment:
i can refactor all classes in plugin on our convention but i think it will not be case in my task. Gradually, all classes will be rewritten on convention of our time)